pkg/model/logstore/logstore.go

```diff
@@ -458,6 +471,10 @@ func (s *LogStore) ManifestLog(mn model.ManifestName) string {
 				continue
 			}
 
+			if !currentSeg.CanContinueLine(segment) {
+				break
```

Comment:
good question! when the logs are appended, we'll correctly mark up the line continuations. i added your example as a test to confirm this.

re: LogInterleaving. Ya, that's right. This is really more of a product decision than a technical one, but the intuition is that:
- All the logs from a single source should appear in the order that they're printed, because that's what the source intended to happen.
- If Tilt is interleaving logs from different sources, it should try to re-order them to preserve the integrity of each source.

In other words: if two different pod logs are getting interleaved by Tilt, then that's the LogStore's problem to sort out. Because who else would fix it? The individual pods can't do anything to fix it. 

If, say, there's a bug in Tilt's ImageBuilder where it's interleaving Info and Debug and Warning logs in a weird way, then we should just fix BuildController.